src/integrationTest/groovy/wooga/gradle/build/UnityBuildPluginIntegrationSpec.groovy

```diff
@@ -19,112 +19,69 @@ package wooga.gradle.build
 
 import org.junit.Rule
 import org.junit.contrib.java.lang.system.EnvironmentVariables
+import org.yaml.snakeyaml.Yaml
+import spock.genesis.Gen
+import spock.genesis.transform.Iterations
+import spock.lang.Shared
 import spock.lang.Unroll
+import wooga.gradle.unity.batchMode.BatchModeFlags
 
 class UnityBuildPluginIntegrationSpec extends UnityIntegrationSpec {
 
-    @Unroll
-    def "can override default platforms/environments in extension with #propertyInstruction"() {
-        given: "a default gradle project"
-
-        assert runTasksWithFailure(taskName, "--dry-run")
-
-        when: "change default platforms"
-        buildFile << """
-            ${propertyInstruction}
-        """.stripIndent()
+    @Rule
+    public final EnvironmentVariables environmentVariables = new EnvironmentVariables()
 
-        then:
-        runTasksSuccessfully(taskName, "--dry-run")
+    def setup() {
+        //create the default location for app configs
+        def assets = new File(projectDir, "Assets")
+        def appConfigsDir = new File(assets, "UnifiedBuildSystem-Assets/AppConfigs")
+        appConfigsDir.mkdirs()
+
+        ['ios_ci', 'android_ci', 'webGL_ci'].collect { createFile("${it}.asset", appConfigsDir) }.each {
+            Yaml yaml = new Yaml()
+            def buildTarget = it.name.split(/_/, 1).first()
+            def appConfig = ['MonoBehaviour': ['bundleId': 'net.wooga.test', 'buildTarget': buildTarget]]
```

Comment:
I've actually named this string `batchModeBuildTarget` so it's not confusing in C# world. 
Because in C# world there's `buildTarget` and `buildTargetGroup` already.
https://github.com/wooga/wdk-unity-UnifiedBuildSystem/pull/16/files#diff-e4164979f5701a601ef92b1b509d68eaR14

Reply:
Yeah, this key basically, sorry, first spotted it in the test https://github.com/wooga/atlas-build-unity/pull/16/files#diff-ad10964d60127f7d0f1edd3c34330c27R31